crates/swift-integration-tests/src/struct_attributes/derive.rs

```diff
@@ -0,0 +1,28 @@
+#[swift_bridge::bridge]
```

Comment:
Awesome tests. Might've been nice to actually use the `Copy/Clone` impls outside of the module down below, but this is fine.